README.md

```diff
@@ -79,13 +79,13 @@ Falls die Startseite ausnahmsweise doch nicht erscheint, so muss mit dem Browser
 Diese besteht nur aus einer Zeile mit einem Link *Go to __configure page__ to change settings* - dort drauf klicken und man kommt zur Einstellungsseite.
 
 Diese hat die folgenden 4 Zeilen:
- * Geiger accesspoint SSID
- Dies ist die SSID des eingebauten APs und kann zwar geändert werden, sollte aber nicht!
- * Geiger accesspoint password
+ * Geiger accesspoint SSID  
```

Comment:
there are trailing blanks at this line and multiple others.

can you configure your editor to strip them?

Reply:
That's not my editor. I want them to be there! If someone reads the MarkDown, it's is easier to read with the spaces. And the Markdown compiler removes them anyway.

Reply:
What exactly is "easier to read" if there are trailing blanks at the end of a text line?

Reply:
OK, i confused trailing with leading - sorry

Reply:
The reason for the two blanks is to get a line break.

Reply:
I still don't get it.

IIRC trailing blanks do nothing at all, except making people who read the source wonder why they were left there.